pkg/distributor/http.go

```diff
@@ -19,10 +19,15 @@ import (
 // PushHandler reads a snappy-compressed proto from the HTTP body.
 func (d *Distributor) PushHandler(w http.ResponseWriter, r *http.Request) {
 	logger := util_log.WithContext(r.Context(), util_log.Logger)
-	userID, _ := tenant.TenantID(r.Context())
-	req, err := push.ParseRequest(logger, userID, r, d.tenantsRetention)
+	tenantID, err := tenant.TenantID(r.Context())
 	if err != nil {
-		if d.tenantConfigs.LogPushRequest(userID) {
+		level.Warn(logger).Log("msg", "error getting tenant id", "err", err)
```

Comment:
should be `level.Error`?

Reply:
I was wondering the same. IMHO a user error should not be an error.

Reply:
I feel like because we return an error http status code this should be level error as well, but I could see the case it's a warn because we handle it. guess it depends what a user will go looking for if they're trying to figure out why the api is returning 400s.

Reply:
Ok. I've changed the level. I think I saw `level.Error` somewhere else.